device/ruijie/x86_64-ruijie_b6510-48vs8cq-r0/systest.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
```

Comment:
Python 2 is deprecated and no longer supported. We are in the process of migrating all Python code in SONiC to Python 3. Please change this shebang to `#!/usr/bin/env python3` and ensure the script is Python 3-compliant.